config/owasp/suppressions.xml

```diff
@@ -12,11 +10,13 @@
     <cve>CVE-2022-22965</cve>
     <cve>CVE-2022-22968</cve>
   </suppress>
-  <suppress>
-    <notes><![CDATA[
-    file name: netty-*-4.1.76.Final.jar
-    ]]></notes>
+  <suppress until="2022-06-20">
+    <notes><![CDATA[ file name: netty-*-4.1.76.Final.jar ]]></notes>
     <packageUrl regex="true">^pkg:maven/io\.netty/netty.*@4\.1\.76\.Final$</packageUrl>
     <cve>CVE-2022-24823</cve>
   </suppress>
+  <suppress until="2022-06-20">
+    <notes><![CDATA[ file name: spring-core-5.3.19.jar ]]></notes>
+    <cve>CVE-2016-1000027</cve>
+  </suppress>
```

Comment:
Can you use instead:
<suppress>
  <notes><![CDATA[
 springframework spring-* packages: https://github.com/spring-projects/spring-framework/issues/24434#issuecomment-1132113566]]></notes>
  <packageUrl regex="true">^pkg:maven/org\.springframework/spring\-.*?@[1-5]\..*$</packageUrl>
  <cve>CVE-2016-1000027</cve>
</suppress>

Reply:
Doesn't seem to work I tried to implement still throws the warning